test/browser/main_test.js

```diff
@@ -40,7 +40,9 @@ describe('main process', () => {
   })
 
   describe('when required', () => {
-    before(() => __require('browser/main'))
+    before(
```

Comment:
Before-hooks wait for returned promises to resolve, so `before(() => import('...'))` should be fine here.